README.md

```diff
@@ -302,6 +302,32 @@ fastify.addHook('preHandler', function (request, reply, done) {
 ```
 Properties from `reply.locals` will override those from `defaultContext`, but not from `data` parameter provided to `reply.view(template, data)` function.
 
+Typing parameters from `reply.locals` in `typescript`: 
+
+```typescript
+interface Locals {
+ appVersion: string,
+ assetsManifest: AssetsManifest,
```

Comment:
Why this addition?

Reply:
Moved to a separate interface for better readability

This option seems less clear:
```typescript
declare module 'fastify' {
  interface FastifyReply {
    locals:
      | Partial<{
        appVersion: string;
        assetsManifest: AssetsManifest;
      }>
      | undefined;
    }
}
```

Reply:
but `AssetsManifest` is not defined in the example it could lead to misreading?

Reply:
Removed undeclared types and added another example with an object
```typescript
interface Locals {
  appVersion: string,
  isAuthorized: boolean,
  user?: {
    id: number
    login: string
  }
}
```